support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.jsx

```diff
@@ -116,11 +117,20 @@ const guardianWeekly: ProductCopy = {
   subtitle: getPrice(GuardianWeekly, ''),
   description: 'A weekly, global magazine from The Guardian, with delivery worldwide',
   offer: getGuardianWeeklyOfferCopy(),
-  buttons: [{
-    ctaButtonText: 'Find out more',
-    link: subsLinks.GuardianWeekly,
-    analyticsTracking: sendTrackingEventsOnClick('weekly_cta', 'GuardianWeekly', abTest),
-  }],
+  buttons: [
+    {
+      ctaButtonText: 'Find out more',
+      link: subsLinks.GuardianWeekly,
+      analyticsTracking: sendTrackingEventsOnClick('weekly_cta', 'GuardianWeekly', abTest),
+    },
+    {
+      ctaButtonText: 'See gift options',
+      link: subsLinks.GuardianWeeklyGift,
+      analyticsTracking: sendTrackingEventsOnClick('weekly_cta_gift', 'GuardianWeekly', abTest),
+      modifierClasses: '',
+      hierarchy: 'second',
```

Comment:
What's the difference between hierarchy and index? I'm not sure I see how hierarchy is used at the moment as we're only checking for `hierarchy='first'` which isn't ever the case yet?

Reply:
Sorry - this is probably a bit opaque from the screen shots I've provided! This is used when there are two different app stores that we link to (on mobile). In this case, the 'equal first' hierarchy overrides the index, which would otherwise make the second button a tertiary rather than secondary button.
![Screen Shot 2019-12-02 at 11 43 54](https://user-images.githubusercontent.com/16781258/69956861-1c27bd00-14f9-11ea-95a8-a3fbf7de57bd.png)

Reply:
And we only check for 'first' so I will remove the one that's 'second'